x-pack/plugins/cross_cluster_replication/public/app/services/auto_follow_pattern.js

```diff
@@ -4,8 +4,15 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+import React, { Fragment } from 'react';
+import { format, addDays } from 'date-fns';
+
 export const getFollowPattern = (prefix = '', suffix = '', template = '{{leader_index}}') => (
-  prefix + template + suffix
+  <Fragment>
```

Comment:
I have a few suggestions!

1. Could this return an object of the shape `{ prefix, suffix, template }` and can we delegate the rendering logic to the `AutoFollowPatternPreview` component? This way we can keep all rendering logic in the `sections` dir.
2. Can we use `<strong>` instead of `<span>`?
3. I'm on the fence, but do you think it might make more sense to bold the index pattern instead, since it is the part that's changing?

![image](https://user-images.githubusercontent.com/1238659/50022287-71895800-ff90-11e8-955a-eaa03f944be8.png)

Reply:
Ok, good points. About where the bold is, this was a request from Yaron. Let me check with him.